docs/DeveloperGuide.md

```diff
@@ -437,6 +486,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
    * 1c2. User re-enters a new command with another field.
    * Steps 1c1 - 1c2 are repeated until a valid field is inputted by the User.
    * Use case resumes from step 2. 
+
+* 1d. PoochPlanner receive wrong command format.
```

Comment:
I think these kind of extensions should be omitted as they overlap with other extensions.

Reply:
On my second thought, yes it might be overlapping with other extensions. I have removed them from use cases